doc/guide/guide-tensor.rst

```diff
@@ -434,5 +433,6 @@ operators:
   [[[2], [2]], [[3], [3]]]
 
 
-
-.. _channel contraction tutorial: https://nbviewer.ipython.org/github/qutip/qutip-notebooks/blob/master/examples/superop-contract.ipynb
+..
```

Comment:
I'm worried it won't be that easy to put the link bank since the text referring to it has been removed. Perhaps we should just leave the link to the v4 notebook and leave a note for the reader and a TODO for us to update it.

Reply:
Opening an issue works. Let's do that.